Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+# Gopkg.toml example
```

Comment:
Do we need all these comments and the `example` line?

Reply:
It was autogenerated.  I can delete it if you like.

Reply:
Deleted